kube/overlays/dev-integration-test/speed-up-worker.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: apps/v1
```

Comment:
what happened to this test?

Reply:
we don't need to speed up latency when it's using the watch api (since this was only decreasing the wait time for the poll interval)